### ISSUE TITLE
Run a same transfer command two times will bring APP to crash

### ISSUE THREAD

**Issue by mackzheng** [dxjia/ffmpeg-jni-sample#5]
my step：
1. recorder video profile : 
                                mProfile = CamcorderProfile.get(CamcorderProfile.QUALITY_480P);
                mProfile.fileFormat = MediaRecorder.OutputFormat.MPEG_4; 
                mProfile.videoCodec = MediaRecorder.VideoEncoder.H264; 
                mProfile.audioCodec = MediaRecorder.AudioEncoder.AAC;  
                                mProfile.videoFrameRate = 15;  
                mProfile.videoBitRate = 1200000; 
2. create video file :  /sdcard/demo.mp4
3. run  ffmpeg -y -i /sdcard/demo.mp4  -vf crop=iw/3:ih:0:0 -strict -2 /sdcard/demo2.mp4

It's perfect for the first  time. and not kill app,  do the second . the app is dead. 

07-03 00:36:52.363  20375-20375/com.demo D/ffmpeg-jni﹕ ffmpeg
07-03 00:36:52.363  20375-20375/com.demo D/ffmpeg-jni﹕ -version
07-03 00:36:52.363  20375-20375/com.demo D/ffmpeg-jni﹕ reset_record done!
07-03 00:36:52.363  20375-20375/com.demo D/ffmpeg-jni﹕ Splitting the commandline.
07-03 00:36:52.363  20375-20375/com.demo D/ffmpeg-jni﹕ Reading option '-version' ...
07-03 00:36:52.363  20375-20375/com.demo D/ffmpeg-jni﹕ matched as option 'version' (show version) with argument ''.
07-03 00:36:52.363  20375-20375/com.demo D/ffmpeg-jni﹕ Finished splitting the commandline.
07-03 00:36:52.363  20375-20375/com.demo D/ffmpeg-jni﹕ Parsing a group of options: global .
07-03 00:36:52.363  20375-20375/com.demo D/ffmpeg-jni﹕ Applying option version (show version) with argument .
07-03 00:36:52.363  20375-20375/com.demo D/ffmpeg-jni﹕ reset_record done!
07-03 00:36:52.363  20375-20375/com.demo D/ffmpeg-jni﹕ Successfully parsed a group of options.
07-03 00:36:52.363  20375-20375/com.demo A/libc﹕ Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 20375 (com.demo)

**Comment by dxjia** [comment]
Run a same transfer command two times will bring APP to crash.

**Comment by dxjia** [comment]
Have bee fixed in v0.1.7, please update your dependency as follow,

```
dependencies {
    compile 'cn.dxjia:ffmpegexecutor:0.1.7'
}
```

If still have problem, feel free to let me know.
